graphql/schema.graphql

```diff
@@ -167,6 +167,17 @@ type Patch {
   taskCount: Int
   baseVersionID: String
   moduleCodeChanges: [ModuleCodeChange!]!
+  project: PatchProject
```

Comment:
I would name this something like selectedBuilds. There is already a concept of a project which contains all the possible configuration options for running tasks

Reply:
this does not represent the selected builds. it represents all variants and tasks for the patch's project